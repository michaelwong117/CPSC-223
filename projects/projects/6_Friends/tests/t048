#!/bin/bash
# Mutual friends with self

trap "/usr/bin/killall -q -u $USER ./Friends 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./Friends ]; then
  echo './Friends is missing or not executable'
  echo './Friends is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./Friends /c/cs223/hw6/Tests/graph_alpha_mini_12.in mutual < /c/cs223/hw6/Tests/t048.in
