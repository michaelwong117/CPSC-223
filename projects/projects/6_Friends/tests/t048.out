A, B have mutual friends C F
B, B have mutual friends A C D E F
C, D have mutual friends B
